resources/assets/coffee/react/profile-page/main.coffee

```diff
@@ -127,6 +129,12 @@ class ProfilePage.Main extends React.PureComponent
   render: =>
     withMePage = @state.userPage.initialRaw.trim() != '' || @props.withEdit
 
+    profileOrder =
```

Comment:
creating clone in all cases and then adding account_standing as needed should be fine.

```
profileOrder = ....slice()
profileOrder.push('...') if ...
```